library

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
If you leave this file as a `.sh`, I believe it'll be more clear that this is something you source, rather than execute directly (including making this file not executable).